pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/UnusedPrivateFieldRule.java

```diff
@@ -53,7 +52,7 @@ public Object visit(ASTClassOrInterfaceDeclaration node, Object data) {
             VariableNameDeclaration decl = entry.getKey();
             AccessNode accessNodeParent = decl.getAccessNodeParent();
             if (!accessNodeParent.isPrivate() || isOK(decl.getImage()) || classHasLombok
-                    || hasLombokAnnotation(accessNodeParent) || hasNeglectAnnotation(decl)) {
+                || ((Annotateable) accessNodeParent).isAnyAnnotationPresent(getProperty(ignoredAnnotationsDescriptor))) {
```

Comment:
The abstract rule class could have a shorthand method for this, so as to write only `hasIgnoredAnnotation((Annotatable) accessNodeParent)` for example